### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by simrin051**

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
1.The size of pr is good
2.The code looks clean and readable
Good to merge

File: docs.css
Comment (parvez-rashid) [inline docs.css]: Using !important is a bad practice. If there are issues with values not getting applied, try to solve by manipulating specificity.

File: docs.css
Comment (parvez-rashid) [inline docs.css]: Same as above, strictly avoid using !important.

File: docs.html
Comment (parvez-rashid) [inline docs.html]: Remove comments before merging.

File: docs.html
Comment (parvez-rashid) [inline docs.html]: Great that you've used `aria-` attribute for better accessibility.

**Review by parvez-rashid — APPROVED** [review]
I've left a few comments, have a look. Overall, the code looks well-structured. Be careful about using the !important property (I've seen this in your quiz project as well) as it is to be avoided unless there is no other option. 

Code can be merged after making suggested changes.

**Pull request merged**
